README.md

```diff
@@ -0,0 +1,40 @@
+# Ô Chap'
+
+Ô Chap' is an e-commerce website for selling all kinds of hats!
+
+## Installation
+
+### Requirements
+
+- Git
+- JDK 1.8 or later
```

Comment:
The project's POM requests [java 11](https://github.com/AMT-2021/AMT/blob/master/pom.xml#L17).